common/statistics/Cargo.toml

```diff
@@ -0,0 +1,16 @@
+# Copyright 2024 - Nym Technologies SA <contact@nymtech.net>
+# SPDX-License-Identifier: Apache-2.0
+
+[package]
+name = "nym-statistics-common"
```

Comment:
The crate seems to be specifically about session stats (and not necessarily tied to gateway stats?). Probably a more precise name would be helpful, like `nym-session-statistics-common`, or `nym-gateway-statistics-common`, or `nym-gateway-statistics-events`, depending on the future plans for the crate. And the directory having the name matching (minus the `nym-` prefix)

Reply:
For now it is only session stats. In the future, I want to add more event types, that will be used by other components across the mono repo. Hence the name is kinda accurate for the end goal of this crate